tests/util/golinter/README.md

```diff
@@ -0,0 +1,53 @@
+# golinter
+
+This golinter checks if our test files follow customerized lint rules.
+
+# Unit tests rules.
+## File name requirement.
+Unit tests should stay in file with suffix "_test.go".
+## Forbidden function calls.
+Unit tests should not contain any of these function calls.
+```bash
```

Comment:
That doesn't make sense - how do we name non-unit tests  ? There are plenty of cases for unit tests to use goroutine - and even Sleep.